.github/workflows/e2e.yml

```diff
@@ -9,6 +9,11 @@ jobs:
   e2e-test:
     runs-on: ubuntu-18.04
     steps:
+      - name: Set up Go 1.18
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Maybe is better to use 1.20 version? since its the one specified in readme

Reply:
you['re right! thanks luis :)

Reply:
actually go-version 1.2 works a bit oddly in GH actions, I will leave it to 1.18 until future updates